classes/widgets/class-suggested-tasks.php

```diff
@@ -163,9 +163,8 @@ public function enqueue_scripts() {
 		$year             = \gmdate( 'Y' );
 		$current_date     = $year . '-' . \gmdate( 'm-d' );
 
-		// TODO: GET params just for testing.
-		$start_date = $year . '-' . ( isset( $_GET['start_date'] ) ? \sanitize_text_field( \wp_unslash( $_GET['start_date'] ) ) : '02-12' );
-		$end_date   = $year . '-' . ( isset( $_GET['end_date'] ) ? \sanitize_text_field( \wp_unslash( $_GET['end_date'] ) ) : '02-16' );
+		$start_date = $year . '-02-12';
+		$end_date   = $year . '-02-16';
 
 		if ( $current_date >= $start_date && $current_date <= $end_date ) {
```

Comment:
This part doesn't make a lot of sense to me...
Should we perhaps make it like this?
```suggestion
		$date_md = (int) \gmdate( 'md' );
		if ( $date_md >= 212 && $date_md <= 216 ) {
```